### ISSUE TITLE
spacebar keybinding doesn't announce change when using screen reader

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#136798]
Steps to Reproduce:

1. Turn on a screen reader
2. Enable Settings Sync
3. highlight an item and hit space to toggle the checkbox:
![image](https://user-images.githubusercontent.com/2644648/141011659-86cf4ac6-6876-4412-89f9-6d24f542241f.png)

Notice nothing gets announced.

**Comment by isidorn** [comment]
Hmm for this make sure that you are using the role [checkbox](https://w3c.github.io/aria/#checkbox) for each list element and that you properly set `aria-checked` on the items.

**Comment by connor4312** [comment]
Verified with VoiceOver on OSX
